troubleshoot/elasticsearch/diagnosing-corrupted-repositories.md

```diff
@@ -4,47 +4,33 @@ mapped_pages:
   - https://www.elastic.co/guide/en/elasticsearch/reference/current/diagnosing-corrupted-repositories.html
 applies_to:
   stack:
-  deployment:
-    eck:
-    ess:
-    ece:
-    self:
 products:
   - id: elasticsearch
 ---
 
 # Diagnose corrupted repositories [diagnosing-corrupted-repositories]
 
-Multiple {{es}} deployments are writing to the same snapshot repository. {{es}} doesn’t support this configuration and only one cluster is allowed to write to the same repository. See [Repository contents](../../deploy-manage/tools/snapshot-and-restore.md#snapshot-repository-contents) for potential side-effects of corruption of the repository contents, which may not be resolved by the following guide. To remedy the situation mark the repository as read-only or remove it from all the other deployments, and re-add (recreate) the repository in the current deployment:
+Multiple {{es}} deployments are writing to the same snapshot repository. {{es}} doesn’t support this configuration and only one cluster is allowed to write to the same repository. Refer to [Repository contents](../../deploy-manage/tools/snapshot-and-restore.md#snapshot-repository-contents) for potential side-effects of corruption of the repository contents, which might not be resolved by the following guide. To remedy the situation mark the repository as read-only or remove it from all the other deployments, and re-add (recreate) the repository in the current deployment.
```

Comment:
Starting the content of this doc with the sentence `Multiple deployments are writing to the same snapshot repository` feels weird, but not sure how to correct it.

Considering the doc is called "Diagnose corrupted repositories", I would probably start saying that a possible cause for a corrupted repo is that multiple clusters or deployments are writing to the same repository, which is unsupported.

I'd recomment to find a way to improve reading flow, but I understand the original doc looked like this already.